cssparse/jvm/src/test/scala/cssparse/TestUtil.scala

```diff
@@ -1,6 +1,6 @@
 package cssparse
 
-import fastparse.all._
+import fastparse.allString._
```

Comment:
Let's leave this as `fastparse.all._` (and the other as `fastparse.noApi._`) for the time being. That's not to say we can't change it, but this change is already going to be huge enough =) 

Keeping the peripheral stuff backwards compatible and the use-site code unchanged lets us focus on the core changes that really matter. Once this huge diff lands, we'll have plenty of time to go back and tweak these things